.github/workflows/build.yml

```diff
@@ -15,6 +15,7 @@ jobs:
             cmake_args: >-
               -DWARNINGS_FATAL=ON
               -DFAAD=ON
+              -DFFMPEG=OFF
```

Comment:
Please update also the debian/rules file with this setting

Reply:
Wait. Then FFmpeg will be disabled for all our Ubuntu packages.

That's why I always argued that the packaging files do not belong in the main repo. We need separate versions for separate distribution releases. It only worked until now because of the hacks in CMakeLists.txt.

Reply:
Ah yes. Unfortunately this issue blocks this PR. 

Maybe it is pragmatic to restore the special case CMakeList.txt and proceed.

This offers the best out of the box behavior for home builds as well.

Reply:
No. It has to be fixed in the `rules`.

I will retarget this PR to `main` or otherwise prefer to close it.

Reply:
Right, we don't

Reply:
Then we can just disable it on Ubuntu and merge this into 2.3?